rest/route/task_routes.go

```diff
@@ -407,10 +407,11 @@ func (trh *taskRestartHandler) ParseAndValidate(ctx context.Context, r *http.Req
 		}
 	}
 	trh.taskId = projCtx.Task.Id
-	if projCtx.Project == nil {
-		return fmt.Errorf("Unable to fetch associated project")
+	project, err := projCtx.GetProject()
+	if err != nil {
```

Comment:
Should also check project = nil here to be consistent with what it was before

Reply:
fixed in response to all your comments, but their is a partial github outage, so I can't view changes to reply to your comments